src/patterns/help-users-to/cookies-settings/index.njk

```diff
@@ -21,7 +21,7 @@ There are two parts to implementing cookies control within a service; cookies ba
     patternlibExample({ "path": "components/cookies-banner/examples/cookies-banner/index.njk" })
 }}
 ## How to use
-The cookies banner must be placed directly above the ONS [header](/components/header) at the very top of the page.
+The cookies banner must be placed directly above the ONS [header](/components/header) at the very top of the page. If using the [page template](/foundations/page-template) it should be placed inside the `preHeader` block. This will ensure that it is displayed before the [skip to content](/components/skip-to-content) link.
```

Comment:
```suggestion
The cookies banner must be placed directly above the ONS [header](/components/header) at the very top of the page. If using the [base page template](/foundations/page-template) it should be placed inside the `preHeader` block. This will ensure that it is displayed before the [skip to content](/components/skip-to-content) link.
```